tests/aws/services/stepfunctions/legacy/test_stepfunctions_legacy.py

```diff
@@ -303,7 +303,7 @@ class TestStateMachine:
     @markers.aws.needs_fixing
     def test_create_choice_state_machine(self, custom_client):
         state_machines_before = custom_client.stepfunctions.list_state_machines()["stateMachines"]
-        role_arn = arns.role_arn("sfn_role", SF_TEST_AWS_ACCOUNT_ID, TEST_AWS_REGION_NAME)
```

Comment:
The third argument of this function was actually called `env=` and was unused within the function.

This is now removed.